cmd/plugins.go

```diff
@@ -0,0 +1,501 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/go-flutter-desktop/hover/internal/fileutils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
```

Comment:
imports need sorting.

Reply:
note to myself: never add newlines in import statements, it messes `gofmt`